### ISSUE TITLE
Change variables to expressions

### ISSUE THREAD

**Issue by tarnold17** [project-pareto/project-pareto#240]
In the strategic model, there are quite a few Pyomo `Var`s which would be better implemented as `Expression`s.

**Comment by Tominapa** [comment]
I have done this work. An issue raised during a meeting questioned whether expression-izing constraints was effective with commercial solvers. I did not have access to Gurobi at the time, so the work stalled. If there is still interest, I can run tests with expressions. Of course, will need to update to incorporate any changes in main.

**Comment by tarnold17** [comment]
@Tominapa thanks for the status update. For now, I am going to take this issue off the July 2023 release board, but I think this is worth revisiting in the near(ish) future.

**Comment by tarnold17** [comment]
@Tominapa do you have any plans to revisit the issue? If not, I think we should consider closing as not planned.

**Comment by Tominapa** [comment]
No, I will not have the time required to return to this in the near future. I have tried this with a number of constraints converted to expressions, but it doesn't always result in improvements. Given those results, and the fact that any gains will be marginal, it's fine to close this. We have plenty of other priorities.
